IntelliTect.Multitool/IPaths.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace IntelliTect.Multitool;
+/// <summary>
+/// Provides normalized paths.
+/// </summary>
+public interface IPaths
+{
+    /// <summary>
+    /// Returns the repository root directory name.
+    /// </summary>
+    /// <param name="CurrentProjectFolderName">The highest level directory name of the current project within the repository.</param>
+    /// <returns></returns>
+    public static string GetDefaultRepoRoot(string CurrentProjectFolderName)
```

Comment:
FWIW my personal opinion is the better way to find the repo root is to look for the .git directory rather than base it on the code contents. Since this is a git repository you know that (outside of things like submodules) it is what git uses to determine the root so it will be more consistent and not likely to move around.